setup.go

```diff
@@ -88,6 +88,11 @@ var primaryFormat LogFormat = ColorizedOutput
 // defaultLevel is the default log level
 var defaultLevel LogLevel = LevelError
 
+// globalLevel is the current global log level
+// New loggers will be set to `defaultLevel` by default but the current global level is currently tracked separately.
+// QUESTION: Do we want to use defaultLevel and override what new loggers will be set to?
+var globalLevel LogLevel = LevelError
+
```

Comment:
@gammazero I vaguely remember you mentioned some limitation of go-log in the regard of inability to change global/default level on the fly - thoughts? (see this and other QUESTION).

Reply:
I think that comment pertained to the `PipeReader` because the level is only set when it is created, [here](https://github.com/ipfs/go-log/blob/master/pipe.go#L55), and reading a different level would require creating a new `PipeReader`

Reply:
fixed by [`43b4c60` (#160)](https://github.com/ipfs/go-log/pull/160/commits/43b4c6014f8dce67a7bd6a05de7185e6d41eca28)